src/CQRS/LeanCode.CQRS.AspNetCore/Registration/CQRSObjectsRegistrationSource.cs

```diff
@@ -2,25 +2,29 @@
 using LeanCode.Components;
 using LeanCode.Contracts;
 using LeanCode.CQRS.Execution;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace LeanCode.CQRS.AspNetCore.Registration;
 
 internal class CQRSObjectsRegistrationSource
 {
-    private readonly TypesCatalog contractsCatalog;
-    private readonly TypesCatalog handlersCatalog;
+    private readonly IServiceCollection services;
 
-    public IEnumerable<CQRSObjectMetadata> Objects { get; }
+    public IEnumerable<CQRSObjectMetadata> Objects { get; private set; }
 
-    public CQRSObjectsRegistrationSource(TypesCatalog contractsCatalog, TypesCatalog handlersCatalog)
+    public CQRSObjectsRegistrationSource(
+        IServiceCollection services,
+        TypesCatalog contractsCatalog,
```

Comment:
I think having this in constructor is redundant, let the caller explicitly call `AddCQRSObjects`